tests/test_jupyter.py

```diff
@@ -10,14 +10,13 @@ class TestJupyter(BaseTmpl):
     def setUp(self):
         super().setUp()
         try:
-            from IPython.testing.globalipapp import start_ipython
-            self.start_ipython = start_ipython
+            from IPython.terminal.interactiveshell import TerminalInteractiveShell
+            self.ip = TerminalInteractiveShell.instance()
```

Comment:
If we have more tests in the future, are we supposed to use this specific instance repeatedly, or we should create a new one? If we are supposed to create a new one, then maybe do not store the instance in the test class, store the create method?